smartGuitar/www/chords/chords.controller.js

```diff
@@ -424,18 +424,26 @@
 
             var scoreLine = "";
             var frets = [0, 0, 0, 0, 0, 0];
+            var note= [0,0];
+            var clicked_notes=0;
 
             for (var fretIdx = 0; fretIdx < vm.score[scoreIdx].length; fretIdx++) {
                 for (var stringIdx = 0; stringIdx < vm.score[scoreIdx][fretIdx].length; stringIdx++) {
                     if (vm.score[scoreIdx][fretIdx][vm.numberOfStrings - stringIdx - 1].show) {
                         scoreLine = scoreLine + "s" + (stringIdx + 1) + "f" + fretIdx + " ";
                         frets[stringIdx] = fretIdx + 1;
+                        note[0]=stringIdx; note[1]= fretIdx+1;
+                        clicked_notes++;
                     }
                 }
-
+            }
+            if (clicked_notes == 1){
+                var extracted_chord = "<" + GuitarPlayer.getSingleNote(note) + ">";
+            }
+            else{
+                var extracted_chord = GuitarPlayer.getChord(frets);
```

Comment:
It itches me a little to have this var declared in two separate places. It works technically, but it would be clearer if the variable was declared before the if-else block, and merely used afterwards.